opencog/guile/SchemeSmob.cc

```diff
@@ -262,6 +283,26 @@ void SchemeSmob::register_procs(void*)
 
 	// Iterators
 	register_proc("cog-map-type",          2, 0, 0, C(ss_map_type));
+
+	// Load scheme files from load-path either set by GUILE_LOAD_PATH or
+	// (add-to-load-path "...")
+	// Load core atom types.
+	// The remaining atom types from the cogserver are in (opencog atom-types)
+	register_proc_from_scm("core_types.scm");
+
+	// Load other grunge too.
+	// Some of these things should probably be modules ...?
+	register_proc_from_scm("config.scm");
+
+	register_proc_from_scm("core-docs.scm");
+
+	register_proc_from_scm("utilities.scm");
+
+	register_proc_from_scm("apply.scm");
+	register_proc_from_scm("av-tv.scm");
+	register_proc_from_scm("file-utils.scm");
+	register_proc_from_scm("debug-trace.scm");
```

Comment:
Ugh. I can see that this works. Its vaguely annoying.  Its a shell-game.  Today, these loads are done in the SCM_PRELOAD section of the opencog.conf file.  All you've done is to move them from the conf file, to here.  However ugly the conf file is, its still easier to use than altering C++ code, and recompiling...